WalletWasabi.Fluent/ViewModels/Wallets/Receive/ReceiveAddressViewModel.cs

```diff
@@ -63,6 +62,18 @@ public ReceiveAddressViewModel(UiContext uiContext, IWalletModel wallet, IAddres
 
 	private IAddress Model { get; }
 
+	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
+	{
+		_wallet.Addresses
+			.Watch(Model.Text)
+			.Where(change => change.Current.IsUsed)
+			.Do(_ => Navigate().Back())
+			.Subscribe()
+			.DisposeWith(_disposables);
```

Comment:
Use the `disposables` that is passed in parameters, you don't need your own one.